pkg/mqtrigger/metrics.go

```diff
@@ -45,6 +45,20 @@ var (
 		},
 		[]string{"trigger_name", "trigger_namespace", "topic", "partition"},
 	)
+	triggerStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "fission_mqt_trigger_status",
+			Help: "Status of a trigger",
+		},
+		[]string{"trigger_name", "trigger_namespace"},
+	)
+	mqtInprocessCount = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "fission_mqt_inprocess",
+			Help: "Total number of mqt in process",
```

Comment:
Total number of MQTs in active processing

Reply:
Done